docs/references/astro/migrating-from-astro-community-sdk.mdx

```diff
@@ -1,5 +1,139 @@
 ---
+title: 'Migrating from the Astro community SDK'
 description: Learn how to migrate from the Astro community SDK to the Clerk Astro SDK.
 ---
 
 # Migrating from the Astro community SDK
+
+In June 2024, we introduced official support for Astro. This minimal guide highlights the breaking changes that were introduced and provides examples on how to resolve them.
+
+
+**Overview:**
+- Install `@clerk/astro`, uninstall `astro-clerk-auth`
+- Update import paths
+- Replace already deprecated APIs
+
+## Installation
+
+Uninstall the community SDK and install the new offical SDK for Astro.
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash filename="terminal"
+    npm uninstall astro-clerk-auth
+    npm install @clerk/astro
+    ```
+
+    ```bash filename="terminal"
+    yarn remove astro-clerk-auth
+    yarn add @clerk/astro
+    ```
+
+    ```bash filename="terminal"
+    pnpm remove astro-clerk-auth
+    pnpm add @clerk/astro
+    ```
+</CodeBlockTabs>
+
+## Update import paths
+We have kept breaking changes to the minimum and there is a great chance that it will not affect your existing codebase.
+As a quick solution find all the usages of `astro-clerk-auth/` and replace them `@clerk/astro/`
```

Comment:
```suggestion
As a quick solution find all the usages of `astro-clerk-auth` and replace them `@clerk/astro`
```
I think this reads a little nicer, with the same effect. Not a required change though.